lib/pages/chat/chat_event_list.dart

```diff
@@ -129,11 +129,18 @@ class ChatEventList extends StatelessWidget {
                       );
                     }
                     if (controller.timeline!.canRequestHistory) {
-                      return Center(
-                        child: IconButton(
-                          onPressed: controller.requestHistory,
-                          icon: const Icon(Icons.refresh_outlined),
-                        ),
+                      return Builder(
+                        builder: (context) {
+                          WidgetsBinding.instance.addPostFrameCallback(
+                            (_) => controller.requestHistory(),
+                          );
```

Comment:
I updated it in desc

Reply:
I'm also worried about the usage of addPostFrameCallback inside of build method. This might cause some unexpected behavior if the widget is rebuilt multiple times

Reply:
I updated for this logic, removed callback inside the build method.

Reply:
Pls review again